runtime/doc/eval.txt

```diff
@@ -4034,6 +4035,19 @@ findfile({name} [, {path} [, {count}]])				*findfile()*
 <		Searches from the directory of the current file upwards until
 		it finds the file "tags.vim".
 
+flatten({list} [, {maxdepth}])					*flatten()*
+		Flatten {list} to depth {maxdepth}.  {maxdepth} means how
+		deeply it makes flatten the list.  the depth is 1 when
```

Comment:
This is not completely clear.  Is it correct to say that the result is the depth of the list, i.e., a depth 1 list contains only Dict, Number, String, etc, whereas a depth 2 list may contain other lists (which do not contain lists themselves)?

Reply:
I'm sorry. My English is not good.

> Is it correct to say that the result is a List having depth N

I already described at the https://github.com/vim/vim/pull/3676#issuecomment-445460589 .
Please check [src/testdir/test_flatten.vim](src/testdir/test_flatten.vim) for more examples.

Thanks for your reviews :smile: